alembic/versions/0490afed87f2_relay_log_entity.py

```diff
@@ -1,16 +1,16 @@
-"""add bouquets table
```

Comment:
Why are you editing a working migration file instead of creating a new one?

Reply:
I think this occurred when we were trying to find the reason why tests are affecting development migrations where I had to delete all migrations that were generated and regenerate them again

Reply:
@Emile-Nsengimana but you have not corrected it? You've removed an old migration. Please undo this and add a new one of your own.